bin/makeLinkFarm

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+from __future__ import print_function
 import argparse, os, re, sys
```

Comment:
Oh I see. I wasn't running flake8 on this file since it's not one we actually use. I'll fix it if you are insisting.